src/overlays/actors/ovl_En_Firefly/z_en_firefly.h

```diff
@@ -6,11 +6,33 @@
 
 struct EnFirefly;
 
+typedef void (*EnFireflyActionFunc)(struct EnFirefly*, GlobalContext*);
+
 typedef struct EnFirefly {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x228];
+    /* 0x014C */ Vec3f bodyPartsPos[3];
+    /* 0x0170 */ SkelAnime skelAnime;
+    /* 0x01B4 */ EnFireflyActionFunc actionFunc;
+    /* 0x01B8 */ u8 auraType;
+    /* 0x01B9 */ u8 onFire;
+    /* 0x01BA */ s16 timer;
+    /* 0x01BC */ s16 targetPitch;
+    /* 0x01BE */ Vec3s limbDrawTable[28];
+    /* 0x0266 */ Vec3s transitionDrawTable[28];
+    /* 0x030E */ char unk_30E[0x2];
+    /* 0x0310 */ f32 maxAltitude;
+    /* 0x0314 */ ColliderJntSph collider;
+    /* 0x0344 */ ColliderJntSphItem colliderItems[1];
 } EnFirefly; // size = 0x0374
 
+typedef enum {
+    /* 0 */ KEESE_FIRE_FLY,
+    /* 1 */ KEESE_FIRE_PERCH,
+    /* 2 */ KEESE_NORMAL_FLY,
+    /* 3 */ KEESE_NORMAL_PERCH,
+    /* 4 */ KEESE_ICE_FLY
+} KeeseParams;
```

Comment:
Since this is specifically about the type of keese, I suggest naming it accordingly
```suggestion
} KeeseType;
```